spec/unit/target/build_settings/pod_target_settings_spec.rb

```diff
@@ -254,6 +249,14 @@ class BuildSettings
             xcconfig.to_hash['GCC_PREPROCESSOR_DEFINITIONS'].should == '$(inherited) COCOAPODS=1 NON_TEST_FLAG=1'
           end
 
+          it 'merges pod target xcconfig settings from subspecs' do
```

Comment:
verified the same test passes on 1.5.3 but not in 1.6.0 (as of 41b56a5d91c2d9e66a29d2dec06b20d8aba71447)